powerline/segments/i3wm.py

```diff
@@ -54,12 +54,44 @@ def workspaces(pl, segment_info, only_show=None, output=None, strip=0):
 		'contents': w['name'][min(len(w['name']), strip):],
```

Comment:
Also now I got your idea behind workspaces and will say that this segment should thus be deprecated (you want to replace it with lister+workspace segment, right?), see 3c1d5c615ce4c364a9b922fd46e573a312b107a8 (specifically you need to see where and how linter was modified).